x-pack/test/functional/services/cases/api.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import pMap from 'p-map';
+import { CasePostRequest } from '../../../../plugins/cases/common/api';
+import { createCase, deleteAllCaseItems } from '../../../cases_api_integration/common/lib/utils';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { generateRandomCaseWithoutConnector } from './helpers';
+
+export function CasesAppAPIServiceProvider({ getService }: FtrProviderContext) {
+  const kbnSupertest = getService('supertest');
+  const es = getService('es');
+
+  return {
+    async createCaseWithData(overwrites: { title?: string } = {}) {
```

Comment:
`createCaseWithData` -> `createCase`

Reply:
I created this specifically because there's already a `createCase` that doesn't accept data.

Reply:
Where is that?

Reply:
In the cases_api_integration helpers